### ISSUE TITLE
Messages appear in LiveChat even after the ticket is closed/delete

### ISSUE THREAD

**Issue by TheJarjar** [freescout-helpdesk/freescout#3038]
Hey,

When someone use LiveChat it open a chat ticket in the mailbox. 
Closing the ticket means the conversation is over. Next time the user ask for something it's a new ticket.
I tried closing the chat in the mailbox and even deleting the ticket in the mailbox but still the user has his complete history available in his live chat. 

Is there a way to fix that? Am I doing something wrong?

**Comment by freescout-helpdesk** [comment]
All messages received from some customer via the LIve Chat always go to the same conversation in FreeScout (even if it's closed) and this can’t be changed.

**Comment by TheJarjar** [comment]
Please do what's necessary to change that. 
Closing a ticket should delete the livechat. Or I get I just lost $15 ;)

**Comment by Rodrigo-Enviou** [comment]
I agree to TheJajar. Another question: Is it possible to configure a sound when a new chat pops up? It would be very helpful for the agent to identify when someone is waiting on the chat.
